src/lib_ccaligner/params.h

```diff
@@ -9,11 +9,13 @@
 
 #include "commons.h"
 
-std::string currentTime();
+#include <cstring>
```

Comment:
Is it really needed? ;)

Reply:
@saurabhshri No ;) It should have been `ctime`...